packages/nimble-components/src/utilities/tests/storybook-declarations.d.ts

```diff
@@ -0,0 +1,3 @@
+// This file is a workaround to allow Nimble to import types that Storybook doesn't export correctly
```

Comment:
Are we going to have an issue to track removing this file? Maybe it would continue to be captured in https://github.com/ni/nimble/issues/1146?

Reply:
Yep, I updated the description on that issue to track this 👍